<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1295</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>381</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Open_MLC_Only</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>241</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>72</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton__GentrySag">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Gentri sag</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton__EpidSag">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>200</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Epid Sag</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton__IzoSize">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>IzoSize</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>410</y>
      <width>291</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Couch</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Couch izo size</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_CouchZize">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Couch izo size</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_CouchZize_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>130</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>80</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Couch izo size</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>321</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Colimator</string>
    </property>
    <widget class="QLabel" name="label_CouchZize_3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GentrySize">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>131</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gentry izo Size</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>231</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Colimator izo size</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>400</y>
      <width>321</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Gentry</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Gentry IZO Size</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GentryIZOSIZE">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Couch izo size</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_CouchZize_5">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>110</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton__ConeIzoSize">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>260</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ConeIzoSize</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton__Summary">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>290</y>
      <width>341</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Summary_MLC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton__SummaryCone">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>150</y>
      <width>341</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ConeSummery</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton__ConeOpen">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>30</y>
      <width>381</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Open_Cone_Only</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slot1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>changeImages()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>630</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton__GentrySag</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>GentrySag()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>535</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton__EpidSag</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>getEpidSag()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>465</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton__IzoSize</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>getIzoSize()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton__ConeIzoSize</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>GetConeIzoSize()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1085</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>548</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton__Summary</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>sumary_Info()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>548</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton__SummaryCone</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>summeryCone()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>960</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>597</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton__ConeOpen</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>OpenCOne()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1090</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>705</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeImages()</slot>
  <slot>GentrySag()</slot>
  <slot>getEpidSag()</slot>
  <slot>getIzoSize()</slot>
  <slot>GetConeIzoSize()</slot>
  <slot>sumary_Info()</slot>
  <slot>summeryCone()</slot>
  <slot>OpenCOne()</slot>
 </slots>
</ui>
